src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -64,6 +65,11 @@ def invoke_package_manager(self, command):
             self.composite_logger.log('[ERROR] Package manager was invoked using: ' + command)
             self.composite_logger.log_warning(" - Return code from package manager: " + str(code))
             self.composite_logger.log_warning(" - Output from package manager: \n|\t" + "\n|\t".join(out.splitlines()))
+
+            process_lock_tree = self.get_process_tree_from_package_manager_msg(out)
+            if process_lock_tree is not None:
+                self.composite_logger.log_warning(" - Process tree of package manager locking process: \n{}".format(str(process_lock_tree)))
```

Comment:
Again, don't mention lock since the process can be for some other error message.

self.composite_logger.log_warning(" - Process tree for the pid in output: \n{}".format(str(process_tree)))